deploy/README.md

```diff
@@ -291,4 +291,32 @@ NO_DELETE_PIPELINE_RESOURCE_GROUP=true
 
 # Attributions And Disclaimers
 
-- [Geotiff file](https://aoigeospatial.blob.core.windows.net/public/samples/sample_4326.tif) provided as sample are attributed to NAIP Imagery available via [Planetary Computer](https://planetarycomputer.microsoft.com) They are covered under [USDA](https://ngda-imagery-geoplatform.hub.arcgis.com)
\ No newline at end of file
+- [Geotiff file](https://aoigeospatial.blob.core.windows.net/public/samples/sample_4326.tif) provided as sample are attributed to NAIP Imagery available via [Planetary Computer](https://planetarycomputer.microsoft.com) They are covered under [USDA](https://ngda-imagery-geoplatform.hub.arcgis.com)
+
+# Using a pre-provisioned Batch Account to provision Batch Account Pool
+
+We added a script [use_pre_provisioned_batch_account.sh](./deploy/use_pre_provisioned_batch_account.sh) to create batch account pool on a pre-provisioned batch account.
+
+To use the script, first deploy your infrastructure as usual and then use [use_pre_provisioned_batch_account.sh](./deploy/use_pre_provisioned_batch_account.sh) as suggested follows.
+
+```bash
+# Execute install.sh script
+./deploy/install.sh <environmentCode> <location> <envTag>
+
+# Execute use_pre_provisioned_batch_account.sh script
+./deploy/use_pre_provisioned_batch_account.sh <environmentCode> \
+    <preProvisionedBatchAccountName> <preProvisionedBatchAccountResourceGroupName> \
```

Comment:
We would like to have batch account in the same region where the data / rest resources are present. Either we modify the README to call that out or we introduce a check to validate the locations are same

Reply:
@mandarinamdar I'm thinking should it be the other way around because in this scenario we will have a batch account provisioned already (Ex. eastus) and if we want to have all the resources in the same region then the rest of the deployment should be based on this region. Correct?

Reply:
Done! basically the script is checking that synapse workspace and batch account are in same location